src/lvl_script_value.c

```diff
@@ -1049,7 +1049,17 @@ void script_process_value(unsigned long var_index, unsigned long plr_range_id, l
             if (operation == SOpr_INCREASE) computed = current_flag_val + sum;
             else if (operation == SOpr_DECREASE) computed = current_flag_val - sum;
             else if (operation == SOpr_MULTIPLY) computed = current_flag_val * sum;
-            else if (operation == SOpr_DIVIDE) computed = current_flag_val / sum;
+            else if (operation == SOpr_DIVIDE)
+            {
+                if(sum == 0)
+                {
+                    WARNMSG("COMPUTE_FLAG tried to divide by 0.");
```

Comment:
If a player creates this in a next_command_reusable, it will still crash the game, but with a 30 minute delay until the log file is too large.